Adding .NET SDK and tests for new RP - Managed Service Identity

varunkch opened the pull request:
<!-- DO NOT DELETE THIS TEMPLATE -->

## Description
This PR is for adding the autorest generated C# SDK and tests for the new RP - Managed Service Identity.
Swagger spec pull request: https://github.com/Azure/azure-rest-api-specs/pull/2020
---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [Done] Please add REST spec PR link to the SDK PR
- [Done] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [Done] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [Done] Title of the pull request is clear and informative.
- [Done] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [Done] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [Done] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

Commit:
Adding .NET SDK and tests for new RP - Managed Service Identity

Commit:
Adding .NET SDK and tests for new RP - Managed Service Identity

Commit:
ServiceBus: Geo DR new Authorization rule API, AlternateName property added and updates (#3948)

* Geo DR new Autho rule API, AlternateName property added and updates

* description updated in proj file

* Session records updated for Subcription CURD

* AzSdk.RP.props

DavidLiCIG committed:
Regenerate SDKs with a missing parameter in AddFaceFromStream operation in FaceList (#3941)

* Regenerate SDKs.

* Remove unnecessary files

Commit:
EventHub : Geo DR Update (#3944)

* Geo DR update

* version updatd

* AzSdk.RP.props

* removed the GenerateAssemblyInfo: false from proj files

shahabhijeet commented on src/SDKs/ManagedServiceIdentity/Management.ManagedServiceIdentity/Microsoft.Azure.Management.ManagedServiceIdentity.csproj:
```diff
@@ -0,0 +1,25 @@
+    <Description>Provides developers with libraries for ManagedSericeIdentity under Azure Resource manager to perform CRUD operations on identities.</Description>
+    <Version>1.0.0.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.Management.ManagedServiceIdentity</AssemblyName>
+    <PackageTags>Microsoft Azure resource management;msi;managedserviceidentity</PackageTags>
```
add ; at end of the last tag

shahabhijeet commented on src/SDKs/ManagedServiceIdentity/ManagedServiceIdentity.Tests/ManagedServiceIdentity.Tests.csproj:
```diff
@@ -0,0 +1,33 @@
+  <!--Do not remove until VS Test Tools fixes #472-->
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
+  </ItemGroup>
```
@varunkch use wildcard to add all json files in your project. see an example
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Compute.Tests/Compute.Tests.csproj#L20

varunkch replied:
Done.

shahabhijeet reviewed (changes requested):
Can you do the following in the below order:
1)	Pull latest from upstream (if you have already done this good)
2)	Execute msbuild.exe build.proj from the repo root
a.	This will update the build tools in the current repo
3)	Generate your SDK
4)	Build you sdk at least one from command line of IDE
a.	Msbuild.exe build.proj /t:build /p:Scope=SDKs\CognitiveService\management
5)	Update the PR

varunkch commented:
I am getting the following error on running msbuild.exe build.proj from the repo root:

"C:\Users\vakuma\Documents\GitHub\Forks\azure-sdk-for-net\build.proj" (default target) (1) ->
(GetLatestBuildTools target) ->
  C:\Users\vakuma\Documents\GitHub\Forks\azure-sdk-for-net\tools\bootstrapTools\bootstrap.targets(6,2): error : The remote server returned an error: (404) Not Found. [C:\Users\vakuma\Documents\GitHub\Forks\azure-sdk-for-net\build.proj]

    0 Warning(s)
    1 Error(s)

Commit:
Addressing review comments:
o Using wildcard to include all json files in SessionRecords
o Adding semicolon at the end of tags in csproj file

Commit:
Addressing review comments:
o Using wildcard to include all json files in SessionRecords
o Adding semicolon at the end of tags in csproj file

Commit:
Merge branch 'msi_sdk_2' of https://github.com/varunkch/azure-sdk-for-net into msi_sdk_2
